### DISCUSSION TITLE
I got the wifi password now what?

### DISCUSSION THREAD

**Discussion by EmpoweredPhoenix** [3kh0/ext-remover#431] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
i followed the steps now what do i do with the password

**Comment by Aka-but-nice** [comment, marked as answer]
Change your dns to either 
1) 5.189.139.56

2) 51.222.236.102

3) 161.97.156.62

4) 72.5.33.65

5) 129.213.138.223

6) 45.32.216.91

7) 45.90.30.118

8) 198.98.53.76/

**Reply by CheezeKake77** [reply]
Does it matter which one you pick?

**Reply by Darcii** [reply]
Ok but HOW do you change the dns with just the wifi password?

**Reply by EmpoweredPhoenix** [reply]
exactly darcii that was what i was asking

**Reply by AS3jupiter** [reply]
how did u get the wifi password? using the method in the readme.md

**Reply by AS3jupiter** [reply]
did u use the method??

**Reply by Darcii** [reply]
Yes obvy

**Reply by AS3jupiter** [reply]
but it doesn't work

**Reply by Aka-but-nice** [reply]
Yes, I did I missed some steps. After changing you're DNS go to chrome://restart

**Reply by thebossjeff** [reply]
BUT HOW DO YOU CHANGE YOUR DNS GOD

**Reply by thebossjeff** [reply]
THATS ALL HES ASKING
